src/lib/y2storage/proposal/autoinst_md_planner.rb

```diff
@@ -156,6 +155,23 @@ def raid_level_from(raid_options)
         issues_list.add(:invalid_value, raid_options.raid_type, :raid_type, "raid1")
         Y2Storage::MdLevel::RAID1
       end
+
+      # MD device to be reused by the given planned MD
+      #
+      # @param md      [Planned::Md] Planned MD RAID
+      def find_md_to_reuse(md)
```

Comment:
NP: only a reminder. In SP1 we have the "alternative_name" mechanism for `Devicegraph#find_by_any_name`. Maybe we could use it.

Reply:
I have implemented this and just found it brings no value, at least with the current implementation of `Devicegraph#find_by_any_name`. For each alternative name all methods are tried before passing to the next one. That means the disk lookup is potentially used even more often than before.

I will try to adapt the implementation of `Devicegraph#find_by_any_name`